apps/web/src/server/app/validation/validation.controller.js

```diff
@@ -1,13 +1,52 @@
 import { findAllNewAppeals } from './validation.service.js';
 
+/**
+ * Given an item object transform it into - govuk/components/table/macro.njk" import govukTable format
+ *
+ * @param {*} item to do
+ * @returns {any} to do
+ */
+function makeGovTableRow(item){
+	return [
+		{ html: '<a href="#">'  + item.AppealReference + '</a>' },
+		{ text: item.Received },
+		{ text: item.AppealSite }
+	];
+}
+
+/**
+ * Main Route Entrey Point
+ *
+ * @param {*} request to do
+ * @param {*} response to do
+ * @returns {void} to do
+ */
 async function getValidationDashboard(request, response) {
+
 	const appealsListData = await findAllNewAppeals();
 
+	const incompleteAppeals = [];
+	const newAppeals = [];
+
+	for (const item in appealsListData ) {
```

Comment:
I would have used reduce. But that is just me :).

Reply:
see previous comments about lint errore
👍

Reply:
Thats ok, you can keep it as is.